src/linear_algebra/basic.lean

```diff
@@ -2213,3 +2213,25 @@ rfl
 end general_linear_group
 
 end linear_map
+
+section
+
+variables [ring R] [add_comm_group M] [module R M] (T : M →ₗ[R] M)
+
+/-- any invertible linear map can be written as a linear equivalence -/
+def linear_equiv.of_invertible [invertible T] : M ≃ₗ[R] M :=
+linear_map.general_linear_group.to_linear_equiv (unit_of_invertible T)
+
+lemma linear_equiv.coe_of_invertible [invertible T] :
+  ↑(linear_equiv.of_invertible T) = T := by { ext, refl }
+
+lemma linear_map.to_equiv_symm_eq_inv_of
```

Comment:
This doesn't match the naming conventions (there is no `to_equiv`; perhaps another reviewer can suggest a better name.

Reply:
Well `linear_map.inv_of_eq_symm_of_invertible` seems like a reasonable name here.